src/components/views/dialogs/UserSettingsDialog.tsx

```diff
@@ -50,6 +50,10 @@ import { EncryptionUserSettingsTab } from "../settings/tabs/user/EncryptionUserS
 interface IProps {
     initialTabId?: UserTab;
     showMsc4108QrCode?: boolean;
+    /**
+     * If true, the flow for a user to reset their encryption will be shown. initialTabId must be UserTab.Encryption.
```

Comment:
Presumably, this "initialTabId must be UserTab.Encryption" bit only applies when `showResetIdentity` is true? It could do with clarifying, if so. (Again: newlines have no semantic meaning in doc-comments)

```suggestion
     * If `true`, the flow for a user to reset their encryption will be shown. In this case, `initialTabId` must be `UserTab.Encryption`.
     *
```